ods_ci/tests/Resources/OCP.resource

```diff
@@ -108,7 +108,7 @@ Get Events And Pod Logs
 
 Wait For Pods To Be Ready
     [Arguments]    ${label_selector}    ${namespace}    ${timeout}=600s    ${exp_replicas}=${NONE}
-    Wait Until Keyword Succeeds    ${timeout}    3s
+    Wait Until Keyword Succeeds    ${timeout}    15s
```

Comment:
better to parametrize it. The retry depends on the timeout as well, this may impact other test cases